src/OTSession.js

```diff
@@ -85,12 +86,14 @@ OTSession.propTypes = {
   eventHandlers: PropTypes.objectOf(PropTypes.func),
   onConnect: PropTypes.func,
   onError: PropTypes.func,
+  options: PropTypes.object,
 };
 
 OTSession.defaultProps = {
   eventHandlers: null,
   onConnect: null,
   onError: null,
+  options: undefined,
```

Comment:
Do you think the default should be an empty object instead?

Reply:
Hum, not sure about this one, but I don't think it really matters.

Reply:
Agree, better use empty object as default. Can you please push the change?